runner/app/routes/image_to_image.py

```diff
@@ -97,7 +99,12 @@ async def image_to_image(
         seeds = [seeds]
 
     output_images = []
-    for img, s in zip(images, seeds):
-        output_images.append({"url": image_to_data_url(img), "seed": s})
+    for img, sd, is_nsfw in zip(images, seeds, has_nsfw_concept):
+        # TODO: Return None once Go codegen tool supports optional properties
```

Comment:
Initially, I returned `None` when users did not set the `safety_check` parameter, but due to limitations in the Go codegen tool, this wasn't feasible. I now return `false` in such cases. However, I believe returning `None` in the future would more accurately indicate that the check was not performed. What are your thoughts on this approach for handling unset parameters?